File: src/main/java/net/dashflight/data/jwt/create/request/DashflightCreateJwtRequestProviderModule.java
package net.dashflight.data.jwt.create.request;

import com.google.inject.AbstractModule;

public class DashflightCreateJwtRequestProviderModule extends AbstractModule {

    @Override
    protected void configure() {
        bind(CreateJwtRequestProvider.class).to(DashflightCreateJwtRequestProvider.class);
    }
}

File: src/main/java/net/dashflight/data/jwt/create/request/DashflightCreateJwtRequestProviderModule.kt
package net.dashflight.data.jwt.create.request

import com.google.inject.AbstractModule

class DashflightCreateJwtRequestProviderModule : AbstractModule() {

    override fun configure() {
        bind(CreateJwtRequestProvider::class.java).to(DashflightCreateJwtRequestProvider::class.java)
    }
}